### DISCUSSION TITLE
How to apply formating

### DISCUSSION THREAD

**Discussion by alfredopalhares** [polarmutex/beancount-language-server#386] [General]
Hello, first of all, thank you I have recently moved to mason for LSP managment, with that, also changed a language server from [vim-beancount](https://github.com/nathangrigg/vim-beancount) and one feature I loved was the `:AlignCommodity` ption.

Is there a way I can minic the same formating with this language server ?

**Comment by polarmutex** [comment]
I have basic formatting built into the Lsp(basically aligns price/commodity)

[calling :lua vim.lsp.buf.format() should invoke it](https://neovim.io/doc/user/lsp.html#vim.lsp.buf.format())

let me know if that works for you. If not reply back with comments and I am open to making changes to how I format
